tensorbay/opendataset/VGGFace2/loader.py

```diff
@@ -0,0 +1,183 @@
+                    category=class_id_name[info[0]],
+                    attributes=class_name_attrs[class_id_name[info[0]]],
+                )
+                data.label.box2d = [LabeledBox2D.from_xywh(*all_box2d_cords[line.rstrip("\n")])]
+                if all_box2d_attributes.get(line.rstrip("\n"), 0):
+                    data.label.box2d[0].attributes = all_box2d_attributes[line.rstrip("\n")]
+                data.label.keypoints2d = [LabeledKeypoints2D(all_keypoints[line.rstrip("\n")])]
+                segment.append(data)
+    return dataset
+
+
+def _get_classification_info(
+    root_path: str, attributes: Generator[str, None, None]
+) -> Tuple[Dict[str, str], Dict[str, Dict[str, object]]]:
+    class_id_name = {}
+    class_name_attrs = {}
+    with open(os.path.join(root_path, "label", "identity_meta.csv"), "r", encoding="utf-8") as fp:
+        lines = islice(csv.reader(fp), 1, None)
+        for meta_identities in lines:
+            # The normal format of each line of the file is
+            # '<class_id>,"<class_name>",<sample_num>,<flag>,<gender>,\n'
+            # but now there is an error type
+            # '<class_id>,"<class_,name>",<sample_num>,<flag>,<gender>\n'
+            if meta_identities[-1] != "":
+                # join the splitted "class_ and name"
+                meta_identities[1] = "".join(meta_identities[1:3])
+                # Starting from the 4th element, each element moves forward one step
+                meta_identities.pop(2)
+                meta_identities[4] = meta_identities[4].rstrip("\n")
+            class_name = meta_identities[1].strip('"')
+            class_id_name[meta_identities[0]] = class_name
+            class_name_attrs[class_name] = dict(
+                zip(
+                    attributes,
+                    (
+                        int(meta_identities[2]),
+                        bool(int(meta_identities[3])),
+                        meta_identities[4],
+                    ),
+                )
+            )
+    return class_id_name, class_name_attrs
+
+
+def _get_box2d_cords(root_path: str) -> Dict[str, Tuple[float, float, float, float]]:
+    all_box2d_cords = {}
+    for file in _BOX2D_FILES:
+        with open(os.path.join(root_path, file), "r", encoding="utf-8") as fp:
+            for box2d_cords in csv.DictReader(fp):
+                box2d_cords["NAME_ID"] = box2d_cords["NAME_ID"].strip('"')
+                all_box2d_cords[f"{box2d_cords['NAME_ID']}.jpg"] = (
+                    float("X"),
+                    float("Y"),
+                    float("W"),
+                    float("H"),
+                )
+    return all_box2d_cords
+
+
+def _get_box2d_attributes(root_path: str) -> DefaultDict[str, AttributeType]:
+    all_box2d_attrs: DefaultDict[str, AttributeType] = defaultdict(dict)
+    box2d_files = glob(os.path.join(root_path, "label", "attributes", "*.txt"))
+    for file in box2d_files:
```

Comment:
I think we need an example about the csv file content, and put them in comments?